python/doc/examples/probabilistic_modeling/quick_start_guide_distributions.ipynb

```diff
@@ -0,0 +1,824 @@
D7fAFxGwAACtz9AHWyQPfLiFI1cwgIAFAGh9Bi+EbPzDLXywBQawyB0AAMAwAhYAAIBhBCwAAADDWIMFAGfy5ptSVpZ0/Li3Kzkr2w+Lsm0szgZ8hs8GrOXLl+vpp5+Ww+HQoEGD9F//9V8aOnRok23XrVunJ598Ut9++62cTqf69OmjX/7yl7r77rtdbSzLUnZ2tl566SUdPXpU119/vVasWKE+ffq01pAA+JusLGn3bm9XAcAP+WTAWrNmjTIzM5Wbm6ukpCQtW7ZMqamp+vrrr9W1a9dG7Tt37qxHH31Uffv2VUhIiN59912lp6era9euSk1NlSQtXrxYzz33nFavXq2ePXvqscceU2pqqr766iuFmbo4HYBz85eLTUruH9c3eFkB4zhyBfgcnwxYS5cu1bRp05Seni5Jys3N1Z/+9CetXLlSjzzySKP2N954o9v9GTNmaPXq1fr444+Vmpoqy7K0bNkyzZ07V2PHjpUkvfLKK4qOjtZbb72lCRMmeHxMAJrgTxebNHhZAeO4rADgc3wuYNXU1GjLli2aPXu2a1tQUJCSk5NVVFR0zudblqX3339fX3/9tZ566ilJ0t69e+VwOJScnOxq17FjRyUlJamoqIiABXiLL19sUuLIEIAL5nMBq6ysTLW1tYqOjnbbHh0drd1nWQtx7NgxxcXFqbq6Wna7Xb/+9a81evRoSZLD4XD1cXqfDY+drrq6WtXV1a775eXlkiSn0ymn03n+A2sldsuSTfVB86QP1ylJp14O0NQ+bejH5Bx5ok5P8Ms69+411q/T6VR+fr5Gjx5t5mKTkoJDQtz691V+OfcG6zT9795f9qfkP+/5vr4fPcHnAtaFioiI0LZt21RRUaGCggJlZmaqV69ejU4fNldOTo7mz5/faPvGjRsVHh7ewmo9J6W6Wu1VHxA3rl/v7XLOauwpt9cbrjU/P99YX56s0yTqrMfcB26dpubeX/an5D/v+VVVVd4uodXZLMuyvF3EqWpqahQeHq61a9dq3Lhxru2TJ0/W0aNH9fbbbzern3vuuUfff/+9NmzYoO+++069e/fWF198ocTERFebESNGKDExUc8++2yj5zd1BCs+Pl5lZWWKjIxswQg9y56QoKCSEtXFxqp23z5vl3NWbkcHamqM9OnxoxiG6vSEQK+Tua8XiHWannt/2Z+S/7znl5eXKyoqSseOHfPpv6Em+dwRrJCQEA0ePFgFBQWugFVXV6eCggJlZGQ0u5+6ujpXQOrZs6diYmJUUFDgCljl5eX69NNPNX369CafHxoaqtDQ0Ebbg4ODjb15e4L1w2JXm83m03WeznStnponf9mngVwncx+4dXpi7n19f/rLe74v1+YpPhewJCkzM1OTJ0/WkCFDNHToUC1btkyVlZWuTxVOmjRJcXFxysnJkVR/Om/IkCHq3bu3qqurtX79er366qtasWKFpPpfvJkzZ2rhwoXq06eP6zINsbGxbkfJAAAATPDJgDV+/HgdPnxYWVlZcjgcSkxMVF5enmuRenFxsYKC/vUtP5WVlbrvvvu0f/9+tW/fXn379tXvfvc7jR8/3tXmoYceUmVlpe69914dPXpUw4cPV15eHtfAAgAAxvncGixfVV5ero4dO/r8+WOre3fZDhyQFRcnmy9ft0dyv3aPoV9Dp9Op9evXa8yYMeYOSXugTo8I8DqZ+x8EYJ3G595f9qf85z3fX/6GmsSXPQMAABhGwAIAADCMgAUAAGAYAQsAAMAwAhYAAIBhBCwAAADDCFgAAACGEbAAAAAMI2ABAAAYRsACAAAwjIAFAABgGAELAADAMAIWAACAYQQsAAAAwwhYAAAAhhGwAAAADCNgAQAAGEbAAgAAMIyABQAAYBgBCwAAwDACFgAAgGEELAAAAMMIWAAAAIYRsAAAAAwjYAEAABhGwAIAADCMgAUAAGCYzwas5cuXKyEhQWFhYUpKStLmzZvP2Pall17SDTfcoE6dOqlTp05KTk5u1H7KlCmy2WxuP2lpaZ4eBgAACEA+GbDWrFmjzMxMZWdna+vWrRo0aJBSU1N16NChJtsXFhZq4sSJ+uCDD1RUVKT4+HilpKTowIEDbu3S0tJ08OBB18/vf//71hgOAAAIMD4ZsJYuXapp06YpPT1d/fr1U25ursLDw7Vy5com27/22mu67777lJiYqL59++o3v/mN6urqVFBQ4NYuNDRUMTExrp9OnTq1xnAAAECAaeftAk5XU1OjLVu2aPbs2a5tQUFBSk5OVlFRUbP6qKqqktPpVOfOnd22FxYWqmvXrurUqZNGjhyphQsX6pJLLmmyj+rqalVXV7vul5eXS5KcTqecTuf5DqvV2C1LNkmWZemkD9cpScGn3Da1Txv6MTlHnqjTEwK9Tua+XiDWaXru/WV/Sv7znu/r+9ETfC5glZWVqba2VtHR0W7bo6OjtXv37mb18fDDDys2NlbJycmubWlpabrtttvUs2dP7dmzR3PmzNHNN9+soqIi2e32Rn3k5ORo/vz5jbZv3LhR4eHh5zmq1pNSXa32qg+IG9ev93Y5ZzX2lNvrDdean59vrC9P1mkSddZj7gO3TlNz7y/7U/Kf9/yqqipvl9DqbJZlWd4u4lQlJSWKi4vTpk2bNGzYMNf2hx56SB9++KE+/fTTsz5/0aJFWrx4sQoLCzVw4MAztvvuu+/Uu3dvvffeexo1alSjx5s6ghUfH6+ysjJFRkZewMhahz0hQUElJaqLjVXtvn3eLuesgkNCXLedNTVG+nQ6ncrPz9fo0aMVHBx87ic0gyfq9IRAr5O5rxeIdZqee3/Zn5L/vOeXl5crKipKx44d8+m/oSb53BGsqKgo2e12lZaWum0vLS1VTEzMWZ+7ZMkSLVq0SO+9995Zw5Uk9erVS1FRUfr222+bDFihoaEKDQ1ttD04ONjYm7cnWDabJMlms/l0naczXaun5slf9mkg18ncB26dnph7X9+f/vKe78u1eYrPLXIPCQnR
```

Comment:
Github suggestion error:
It is sometimes convenient to create a plot presenting the PDF and CDF on the same graphics. This is possible thanks to Matplotlib.

Reply:
Ok. Fixed.